java/src/test/java/ai/rapids/cudf/TableTest.java

```diff
@@ -326,6 +328,52 @@ void testReadJSONFile() {
       assertTablesAreEqual(expected, table);
     }
   }
+  @Test
```

Comment:
nit: add line to separate tests

```suggestion

  @Test
```